src/styles/images/index.md.njk

```diff
@@ -25,10 +25,30 @@ If your image represents something physical, such as a letter, document or credi
 
 {{ example({group: "styles", item: "images", example: "representative", html: false, open: false, size: "l"}) }}
 
+
 ## Alt text
 
-All images must have an alt text. You should include a description of the image inside the tag to help users who are unable to see it.
+All images must have alt text. This is a description of an image that tells people what information the image provides. It is read out by screen readers or displayed if an image does not load or if images have been switched off.
+
+Alt text must:
+
+- describe the content and function of the image
+- be specific, meaningful and concise
+- use normal punctuation and end with a full stop
```

Comment:
Why end with a full stop? I've never heard that before.

Reply:
Do any of the articles mention why to do that, though?
I've just found this [tip from the W3C](https://www.w3.org/WAI/tutorials/images/tips/):

> As for any text, using punctuation in the text alternative makes the information easier to understand.

I guess it often makes sense. Although I don't think it is a "must". For example, alt texts for icons would be weird as a sentence.